fixieai/client/agent.py

```diff
@@ -44,49 +44,49 @@ def agent_id(self) -> Optional[str]:
         """Return the agentId for this Agent."""
         if self._metadata is None:
             return None
-        return self._metadata["agentId"]
+        return self._metadata["agentId"]  # type: ignore[no-any-return]
```

Comment:
consider doing an assertion on the dtype instead for these
`assert isinstance(self._metadata["agentId"], str)